src/test/java/org/folio/rest/impl/EholdingsTagsImplTest.java

```diff
@@ -125,6 +130,70 @@ public void shouldFailOnInvalidRecordTypeOnGet() {
     assertThat(error.getErrors().get(0).getTitle(), containsString("Invalid 'filter[rectype]' parameter value"));
   }
 
+  @Test
+  public void shouldReturnAllUniqueTags() {
+    insertTags(UNIQUE_TAGS, vertx);
+
+    try {
+      TagUniqueCollection col = getWithOk("eholdings/tags/summary").as(TagUniqueCollection.class);
+
+      assertEquals(col.getData().size(), 4);
```

Comment:
It would be good if tests also checked that response contains correct tag values